easybuild/easyconfigs/p/PaStiX/PaStiX-6.0.1-foss-2018a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'PaStiX'
+version = '6.0.1'
+
+homepage = 'https://pastix.gforge.inria.fr/'
+description = """PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance
+ parallel solver for very large sparse linear systems based on direct methods."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/37632/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['276fff4e95ef0bad1402a7ad40d558722c509e3cc0193fbe509887c639227a30']
+
+dependencies = [
+    ('hwloc', '1.11.8'),
+    ('SCOTCH', '6.0.5'),
```

Comment:
@SimonPinches oughtn't this to be `6.0.6`? `6.0.5` doesn't exist for that toolchain.

Reply:
@verdurin This EB file got accidentally caught up in this PR and was not ready for inclusion.  I've therefore now removed it.